### ISSUE TITLE
BF vocals aren't playing

### ISSUE THREAD

**Issue by TheKamboy** [JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag#460]
### Describe your bug here. If you are modding, try testing your bug in a clean version of the engine instead. Also, if you're using an older version of JSE, please try the latest version/action build. Also, be sure to check the pinned 'JS Engine: Known Issues' issue, to check if your issue hasn't already been found!

Basically if you load a song that has the vocals split between opponent and player it won't play the player side.

Weird thing is, it was working at first.

Screen record too big so here is drive link
https://drive.google.com/file/d/19YCBNo3Zhw9sRj3HPdYr48EsrDJ5Q7-h/view?usp=sharing

### Have you identified any steps to reproduce the bug? If so, please describe them below in as much detail as possible. Use images if possible.

```bash
Basically open a song that has two vocal files.
```

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Linux

### Did you edit anything in this build? If so, mention or summarize your changes.

no

### If you use 1.12.0 or earlier, did you have Optimized Chart Loading turned on?

None

### Did you check for any similar issues to what you're reporting? Check, then come back here. If there is a similar issue, then do not report the issue, otherwise it will be marked as a duplicate.

Yes

**Comment by JordanSantiagoYT** [comment]
you're using a modified bf json. you have to go to the character editor and add the "bf" vocals postfix.

**Comment by TheKamboy** [comment]
> you're using a modified bf json. you have to go to the character editor and add the "bf" vocals postfix.

I actually did test without the custom json and it didn't work but I will test again when I can.

**Comment by TheKamboy** [comment]
Alright, you were right. Thanks for the help!
